### PR TITLE
Add SIG Automation charter and subprojects

### PR DESCRIPTION
**Pull request by johnbelamaric**

### CODE REVIEW

**Comment by johnbelamaric** [comment]
cc @tliron @henderiw @s3wong @n2vo @gvbalaji @vjayaramrh

File: sig-automation/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+- Intent Design
+  - Intent Design
+  - Intent Specialization
+  - Intent Validation
+- Intent Deployment
+  - Cluster Selection
+  - Cross-Cluster Dependencies
+
+In terms of package lifecycle, this includes everything up to and including "fan
+out", where the initial instances of per-cluster packages have been created and
```
Comment (adetalhouet) [inline sig-automation/README.md:45]: How about linking the "fanout" terminology to our glossary: https://github.com/nephio-project/docs/blob/main/glossary.md#fanout ?
Same could be applicable to other specific terms introduced by Nephio present in this document.

WDYT?

File: sig-automation/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+have had basic variance injected by the PackageVariant controller. It does not
+include additional per-cluster specialization done after fan out.
+
+In terms of known R1 components, this subproject will focus on:
+- Porch and Porch enhancements (PackageVariant, PackageVariantSet)
```
Comment (adetalhouet) [inline sig-automation/README.md:50]: Will this live in Nephio's repo, or rather is an enhancements in an upstream project consumed by Nephio?

File: sig-automation/README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  - Resource Actuation
+
+In terms of package lifecycle, this includes everything that happens after the
+package is approved in the deployment repository.
+
+In terms of known R1 components, this subproject will focus on:
+- ConfigSync in the workload clusters
+- free5gc Operator
+- NAD configuration
+- Other node-level network or kernel configurtion
```
Comment (adetalhouet) [inline sig-automation/README.md:127]: typo: configuration

**Review by adetalhouet — COMMENTED** [review]
couple of questions, else LGTM

File: sig-automation/README.md (hunk 1 above)
Comment (johnbelamaric) [reply sig-automation/README.md:45]: yes, good idea, will do

File: sig-automation/README.md (hunk 2 above)
Comment (johnbelamaric) [reply sig-automation/README.md:50]: this bullet is "enhancements to the upstream project" will clarify that

Comment (johnbelamaric) [reply sig-automation/README.md:50]: I seem to have accidentally deleted a few lines around topology controllers and stuff, I will add that back and also add related skills. there should be real telco knowledge needed in this team too

File: sig-automation/README.md (hunk 1 above)
Comment (johnbelamaric) [reply sig-automation/README.md]: I linked to them in the "Primary Skills" lists but not for every occurrence of the words. Let me know if you see some I missed!

File: sig-automation/README.md (hunk 2 above)
Comment (johnbelamaric) [reply sig-automation/README.md:52]: Some of both, updated.

**Review by gvbalaji — APPROVED** [review]
Looks good to me. I have one question . Are CRDs handles by individual teams or is it a collective effort?

File: sig-automation/README.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+**Vice-chair**: Wim Henderickx, Nokia, @henderiw
+
+## Charter
+
+As agreed to by the TSC and documented in the approved [Nephio Community
+Document](https://docs.google.com/document/d/1agsjCN3aCjgPftO8AL4sCkv3Dp2ZRZ82-ikuVHAJTL8/edit?usp=sharing),
+this SIG group focuses on design and standardization of schemas, CRDs, and operators; development of CRDs, Operators, related tooling & reference Implementation; and packaging and testing of functional components of Nephio.
+
+## Subprojects
```
Comment (ganchandrasekaran) [inline sig-automation/README.md:12]: Please correct me, as I recall from during the meeting mins "User Interface" e.g. GUI was planned to be part of a Sub Project. Can we reflect them in here?

Comment (henderiw) [reply sig-automation/README.md:12]: it is called User Interaction Layer and part of Package Management

**Comment by henderiw** [comment]
CRD should be handled by individual teams but we probably need to have the overall group look at them. Going fwd we probably need a team looking at them holistically but I felt for R1 it is too much for now. That does not mean it should stay like this

File: sig-automation/README.md (hunk 4 above)
Comment (ganchandrasekaran) [reply sig-automation/README.md:12]: Ah, Thanks. 
In its respective "Primary Skills" sub-section can we add the desired UI framework/Technology in-plan to use?

Comment (henderiw) [reply sig-automation/README.md:12]: Agreed good point

**Comment by ganchandrasekaran** [comment]
I see previous Contributors-and-committers list here: https://github.com/nephio-project/governance/blob/main/CONTRIBUTORS.md#contributors-and-committers

Based on our survey, is there any plan to make similar table?

**Review by adetalhouet — APPROVED** [review]

**Review by vjayaramrh — APPROVED** [review]

File: sig-automation/README.md (hunk 4 above)
Comment (johnbelamaric) [reply sig-automation/README.md:12]: Ok will do so. BTW, the text above is the TSC approved scope for the SIG overall, I can't really change that language.

**Comment by johnbelamaric** [comment]
@gvbalaji I will add a paragraph echoing what Wim said.

@ganchandrasekaran No, those sort of static tables don't age well. Instead, I suggest we use GH teams for each subproject, if necessary. But just having participants join the GH Org is all we really need to assign issues.

Prow supports automatic assignment and more granular approval controls via OWNERS files. for that, we will likely need to define a set of approvers for each team, and others can go in reviewers.

**Pull request merged**